### ISSUE TITLE
Plots of Land

### ISSUE THREAD

**Issue by alentoghostflame** [MyEssentials/MyTown2#155]
For people that want a big town with many people, trusting everyone in that town, and have them be able to access everyone's stuff is kinda hard for me to do, and very risky.

I would love it if there was a way to not let regular residents build/destroy stuff normally, but you can set it so that person can build in a plot in that city/town. Maybe have it as a setting that you have to do manually, but please add it. If its already added, just tell me the commands then close this     =D

TL;DR; just read it, its not that long.

**Comment by AfterWind** [comment]
It's already added, will be in the next release. /t perm set restrictedTownPerms true
It's a simple flag found in the town itself.
